app/components/search/SearchResults/SearchResults.module.scss

```diff
@@ -179,3 +157,60 @@
     display: block;
   }
 }
+
+@media print {
+  // This below styles are for printing out the results on our service/search results page.
+
+  // This sets visibility hidden on all elements except for the searchResultsContainer
+  // and its children to avoid printing irrelevant sections of the page.
+  // This follows the example of this answer https://stackoverflow.com/a/2618980
+  body {
+    visibility: hidden;
+  }
```

Comment:
This is kind of dangerous because it's not scoped to just this page, so it actually impacts the print view of _all_ pages on the site. I confirmed this by attempting to print the Home Page, which is totally blank in the print view. I think the general approach in the StackOverflow link you wrote in the comment is probably fine, but it was probably written for an era before SPAs, where you could just have the backend serve a different CSS file per page.

I think one way we could still use this approach but scope stuff to just this page would be to attach a custom CSS class to the `<body>` element. Since `<body>` is outside of the React root element, you'll have to use a React Effect to mutate the `<body>` element, and you'll have to remember to undo the CSS class when the component unloads, but it should be pretty easy to do. Here's an article showing how to use the `useEffect` hook to add CSS class to the `body` on component load and remove it on component unload: https://bobbyhadz.com/blog/react-set-body-style. Once you do this, then I think we can change this CSS selector from `body` to whatever class name you chose (note that since we're using CSS modules, you'll probably need to get the class name from the `styles` object so that you have the mangled name).

I'm not sure if this is the best way of doing it, but I think it should be safe, since React Hooks should guarantee that we run the effect on load and unload.

Reply:
I did a bit of research and couldn't find anything better than `useEffect` so I went with that. Thanks for the suggestion!